notebooks/tools/auto-shutdown/ashutdown

```diff
@@ -62,7 +62,8 @@ function check_cpu(){
    else
     # Selects only the final lines (the ones starting with Average: and, among those, only select those whose
     # second column is numeric. For those lines, the third column (cpu load) is printed.
-    CPU_PERCENT=$(mpstat -P ALL 1 1 | awk '/Average:/ && $2 ~ /[0-9]/ {print $3}')
+    # For multicore AI Notebooks we select CPU_PERCENT for the core with most CPU usage
```

Comment:
Thanks for the fix. Would it also makes sense to average all cores CPU usage and use that value?

Reply:
I was thinking about that as well. In our case we had a 16 core AI Notebook. Let us assume that 1 CPU was using 50% of CPU and rest of the cores were not being used. Let us assume my threshold for shutting down the AI Notebook was 5%. If I do the average 50% over 16 cores it will give me 3.1%. I did not want to shutdown the notebook if one of my CPU is being used so heavily. Let me know if you still prefer taking average.

Reply:
What you mentioned makes sense. I will approve for now and we can optimize it in future if needed. Thanks again